src/uu/chgrp/src/chgrp.rs

```diff
@@ -10,20 +10,17 @@
 use uucore::display::Quotable;
 pub use uucore::entries;
 use uucore::error::{FromIo, UResult, USimpleError};
-use uucore::format_usage;
 use uucore::perms::{chown_base, options, IfFrom};
+use uucore::{format_usage, help_about, help_usage};
 
 use clap::{Arg, ArgAction, ArgMatches, Command};
 
 use std::fs;
 use std::os::unix::fs::MetadataExt;
 
-static ABOUT: &str = "Change the group of each FILE to GROUP.";
 static VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
@sylvestre why is `VERSION` only present for some utils and not others.  Shouldnt we use `clap::crate_version` everywhere for uniformity?

Reply:
yeah, probably legacy, we should be uniform

Reply:
don't hesitate to do it :)